src/models/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface RuleDefinition {
+  name: string;
+  description: string;
+  resourceGraph?: ResourceGraphQueryDefinition;
```

Comment:
This works great for now. When you add tests, can we also add a DummyRule type that can always return true so we know what the system kind of looks like w/ multiple rule types?

Might be worth reading the TS docs on discriminated unions if you haven't yet: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html?ref=hackernoon.com#discriminating-unions

Reply:
Adding a DummyRule type sounds like a good idea. Thanks for the link